test/uk/gov/hmrc/tai/service/AutoUpdatePayServiceSpec.scala

```diff
@@ -2608,6 +2608,11 @@ class AutoUpdatePayServiceSpec extends PlaySpec with MockitoSugar {
     else
       Some(200625)
 
+  val midPointofTaxYearPlusOneDay = if (new LocalDate(TaxYear().year + 1,1,1).year().isLeap) {
+    new LocalDate(CurrentYear, 10, 7)
+  } else {
+    new LocalDate(CurrentYear, 10, 6)
+  }
 
```

Comment:
I think joda will give you this number but I'm not sure how to grab it without first checking the API, if you can't find it I'll have a look with you